src/restic/hardlinks_index_test.go

```diff
@@ -0,0 +1,40 @@
+package restic_test
+
+import (
+	"testing"
+
+	"restic"
+	. "restic/test"
+)
+
+// TestHardLinks contains various tests for the HardlinkIndex
+func TestHardLinks(t *testing.T) {
+
+	idx := restic.NewHardlinkIndex()
+
+	idx.AddLink(1, 2, "inode1-file1-on-device2")
+	idx.AddLink(2, 3, "inode2-file2-on-device3")
+
+	var sresult string
+	sresult = idx.GetLinkName(1, 2)
+	Assert(t, sresult == "inode1-file1-on-device2",
```

Comment:
Use `Equals()` instead of `Assert()`, sample is here: https://github.com/restic/restic/blob/master/src/restic/repository/index_test.go#L75